app/lib/admin/metrics/dimension/space_usage_dimension.rb

```diff
@@ -11,7 +11,7 @@ def key
   protected
 
   def perform_query
-    [postgresql_size, redis_size, media_size]
+    [postgresql_size, redis_size, media_size, search_size]
```

Comment:
This doesn't correctly handle the case where Elasticsearch is disabled.

```suggestion
    [postgresql_size, redis_size, media_size, search_size].compact
```